.gitignore

```diff
@@ -1,21 +1,24 @@
+*.swp
```

Comment:
In general I prefer to leave this file alone and make changes to `~/.gitignore_global` instead. I think that mostly you changed some things from one place to another, this creates unnecessary git diff.